tutorials/10_async.md

```diff
@@ -1,13 +1,43 @@
-Generátory a AsyncIO
-====================
+# Generátory a AsyncIO
 
 Výukové materiály:
 [naucse.python.cz](http://naucse.python.cz/2017/mipyt-zima/intro/async/),
 [GitHub](https://github.com/pyvec/naucse.python.cz/tree/master/lessons/intro/async).
 
-Úkol
-====
+## Úkol
 
-Vaším úkolem za 5 bodů bude předělat první úlohu pomocí asyncio.
+Vaším úkolem za 5 bodů je implementovat funkci `gather_acquaintances`, která pomocí asynchronní komunikace s [GitHub API] (včetně [GitHub Search API]) ze všech* issues podle zadané specifikace zjistí všechny uživatele, kteří tyto issue komentovali, a to včetně počtu komentářů. Tato funkce nebude mít asynchronní API (tedy nebude `async def`, ale jen `def`).
```

Comment:
Znamená to, že `gather_acquaintances` má spustit event loop?
Má řešit situaci, kdy je event loop už puštěný?

Reply:
ano. nijak (ale klidně by to mohlo fungovat tak, že ho použije, když existuje)

Reply:
To není dobrý design. Integrace do "možná běžícího" event loopu je křehký hack; předpokládat že loop neběží je zbytečné omezení.
Přidal bych disclaimer:

```
Tato funkce nebude mít asynchronní API, tedy nebude `async def`, ale jen `def`.
Funkce spustí smyčku událostí. Můžete předpokládat, že když je funkce zavolána,
*event loop* neběží.
(Toto je jen pro zjednodušení úkolu a jeho opravování; obecně není dobré kombinovat
logiku a ovládání smyčky událostí do jedné funkce.)
```

Omlouvám se že mi tenhle PR zapadl, takže už nemusí být dost času měnit asyu.

Reply:
https://github.com/cvut/MI-PYT/pull/123/commits/ba92e9569908f5dc2d4f04c8e4bd16e1ae0f065a